AmplifyPlugins/Core/AWSPluginsCore/Model/Support/QueryPredicate+GraphQL.swift

```diff
@@ -61,7 +61,7 @@ extension QueryPredicate {
 
 extension QueryPredicateOperation: GraphQLFilterConvertible {
     var graphQLFilter: GraphQLFilter {
-        return [self.field: [self.operator.graphQLOperator: self.operator.value]]
+        return [field: [self.operator.graphQLOperator: self.operator.value]]
```

Comment:
Are the rest of the `self`s required?

Reply:
hmm.. Xcode removed it automatically. I didnot do this